package.json

```diff
@@ -24,5 +25,9 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
+    "react-router-dom": "^4.3.1",
```

Comment:
Why are these development dependencies? Wouldn't the production build need to use these?